_kosmorro/main.py

```diff
@@ -95,12 +95,15 @@ def main():
         timezone = 0
 
     try:
+        env_no_colors = os.environ.get("NO_COLORS")
+        use_colors = False if env_no_colors is not None else args.colors
```

Comment:
The `in` syntax can simplify the implementation:

```python
        use_colors = "NO_COLOR" in os.environ or args.colors
```


Note that the environment variable's name is `NO_COLOR` without leading `S` in [the standard](https://no-color.org/).

Reply:
I tried adding something like this but it was still printing in color since `"NO_COLOR in os.environ` returns true when the variable is set. Let me know if my changes still need work.

Reply:
I also changed any instances of `NO_COLORS` to `NO_COLOR`!

Reply:
> I tried adding something like this but it was still printing in color since `"NO_COLOR" in os.environ` returns true when the variable is set. Let me know if my changes still need work.

Yeah, my code proposal was not correct, you're right :smile: 
Should have been:

```python
use_colors = not environment.NO_COLOR and args.colors
```

This way, the colors will be displayed if and only if there is no `NO_COLOR` in the env vars, _and_ the user didn't ask to remove the colors with the arguments.